gcp/deployment-packages/autoscale-byol/README.md

```diff
@@ -0,0 +1,130 @@
+# GCP Deployment Manager package for Check Point Autoscaling BYOL solution
+This directory contains CloudGuard IaaS deployment package for Check Point Autoscaling (BYOL) solution published in the [GCP Marketplace](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/checkpoint.vsec?tab=Overview).
+
+# How to deploy the package manually
+To deploy the Deployment Manager's package manually, without using the GCP Marketplace, follow these instructions:
+1. [Download](https://github.com/CheckPointSW/CloudGuardIaaS/archive/master.zip) the CloudGuardIaaS repository
+2. Browse to the CloudGuardIaaS/gcp/deployment-packages/autoscale-byol/ directory
+3. Fill variables in the test_config.yaml(see below for variables descriptions).
+4. Log in into [Google Cloud Platform](https://console.cloud.google.com)
+5. [Activate Cloud Shell](https://cloud.google.com/shell/docs/using-cloud-shell)
+6. Set your Cloud Platform project in the Cloud Shell session use:
+        
+        gcloud config set project [PROJECT_ID]
+7. Upload content of the CloudGuardIaaS/gcp/deployment-packages/autoscale-byol/ directory  to the [cloud shell](https://cloud.google.com/shell/docs/uploading-and-downloading-files) 
```

Comment:
1. Upload **the** content of the ...
2. double space between "directory" and "to"

Reply:
Fixed